common/sass/styles/overrides/react-select.scss

```diff
@@ -0,0 +1,62 @@
+// This syntax is necessary to override css styles in react-select
+.Select {
+  font-size: 1rem;
+  .Select-control {
+    box-sizing: content-box;
+    font-weight: 400;
+    border-radius: 0px;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+    border: 1px solid #ececec;
+    transition: all 120ms;
```

Comment:
Avoid using `all`, we don't want to animate unperformant / position based things like margin, padding, width, height etc. which can sometimes happen on resize. You can use the variable `$transition` which links / buttons use.